composer/datasets/streaming/writer.py

```diff
@@ -12,9 +12,11 @@
 
 import numpy as np
 from tqdm import tqdm
+from upload import get_object_store
```

Comment:
Nit: Use absolute imports
```suggestion
from composer.datasets.streaming.upload import get_object_store
```